paasta_tools/setup_kubernetes_cr.py

```diff
@@ -275,7 +276,9 @@ def format_custom_resource(
     url = get_dashboard_base_url(kind, cluster)
     if url:
         resource["metadata"]["annotations"][paasta_prefixed("dashboard_base_url")] = url
-
+    owner = CRDS_KIND_TO_OWNER[kind.lower()]
```

Comment:
I think we'll get a `KeyError` if `kind.lower()` is not present in the dict. Using `get()` should fix that.

```suggestion
    owner = CRDS_KIND_TO_OWNER.get(kind.lower())
```

Reply:
Great spot!